lib/keycloak/fake_client.rb

```diff
@@ -1,8 +1,10 @@
 module Keycloak
   class FakeClient
-    def initialize
+    def initialize(id: 'foo-bar-id')
+      @id = id
       reset!
     end
+    attr_accessor :id
```

Comment:
Why we need that? Why public? Could be `private attr_reader :id`

Reply:
It's important to set this ID, as keycloak returns it for every user created.

Will move it to a private attribute.